style/pi-hole.css

```diff
@@ -1609,3 +1612,13 @@ textarea.field-sizing-content {
   opacity: 0;
   transition: opacity 200ms ease-in-out;
 }
+
+#StaticDHCPTable td.table-danger {
+  background-color: #ff000022 !important;
+  color: #fff !important;
+}
+
+#StaticDHCPTable td.table-success {
+  background-color: #74c70022 !important;
+  color: #fff !important;
+}
```

Comment:
Quick suggestion just to improve the colors:

```suggestion
#StaticDHCPTable .table-danger {
  background-color: #d337 !important;
}

#StaticDHCPTable .table-success {
  background-color: #2c24 !important;
}
```

Still using semi-transparent colors, but now the contrast is good enough when using the original text color, so I think we don't need to change `color:`.